edx_event_bus_kafka/internal/consumer.py

```diff
@@ -141,8 +137,6 @@ def emit_signals_from_message(self, msg):
         """
         headers = msg.headers() or []  # treat None as []
 
-        # TODO (EventBus): iterate on error handling for missing or multiple event_type headers
-        #  (headers() is actually a list of (key, value) tuples rather than a dictionary)
```

Comment:
Not sure if these need to be code comments, or notes in the consumer error ticket, but:
1. Do we have the offset or other information we can add to error messages?
2. For multiple event type headers, it would be useful to add them to the error message.
3. Maybe we should "Signal not sent.", or something like that to the message?
4. Separate question: where are we committing the offset?

Reply:
1–3, added to https://github.com/edx/edx-arch-experiments/issues/55

4: We aren't committing the offset manually, and it's being done on a timer, which isn't great. https://github.com/edx/edx-arch-experiments/issues/55 includes a note on that.